samples/subsys/ipc/openamp_rsc_table/boards/imx8ulp_evk_mimx8ud7_adsp.conf

```diff
@@ -0,0 +1,8 @@
+CONFIG_LOG_PRINTK=n
+CONFIG_OPENAMP_WITH_DCACHE=y
+
+CONFIG_MBOX=y
+CONFIG_MBOX_INIT_PRIORITY=3
+
+CONFIG_IPM_MBOX=y
+CONFIG_MBOX_NXP_IMX_MU=y
```

Comment:
Question: is it needed? for platform specific driver selection, in general it would depend on its prensence on the DTS which would triger something like:

DT_HAS_NXP_IMX_MU_ENABLED

that is a dependency for the particular peripheral. Or is that not true for the IMX_MU peripheral for the NXP platforms?

Reply:
You're right, this is not needed - https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/mbox/Kconfig.nxp_imx#L7C13-L7C43.
I will remove it.

Reply:
Done!